extension/src/cli/git/constants.ts

```diff
@@ -33,7 +33,7 @@ export enum Flag {
   NAME_ONLY = '--name-only',
   NO_EMPTY_DIRECTORY = '--no-empty-directory',
   NUMBER = '-n',
-  PRETTY_FORMAT_COMMIT_MESSAGE = '--pretty=format:%H %B',
+  PRETTY_FORMAT_COMMIT_MESSAGE = '--pretty=format:%H%n%an%n%ar%nrefNames:%D%nmessage:%B',
```

Comment:
I added a "key name" to get tags (`%D`) and the message (`%B`) since `%D` can possibly output an empty string and the message can by multiple lines.